scripts/bash_completion/optirun

```diff
@@ -54,7 +54,8 @@ _optirun() {
           *)
```

Comment:
Right, I was thinking of something like this:

    --vgl-options)
      ;;

no need to suggest any options at all. Line 19 also needs changes I guess.

While at it, could you fix the tab -> spaces indentation before the `-l|--ldpath|--primus-ldpath)`,the following `;;` line and `--silent -q --verbose`?